dockerfile_scripts/install_deepspeed.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEBIAN_FRONTEND=noninteractive apt-get install -y pdsh libaio-dev
+if [ "$DEEPSPEED_PIP" ]; then
```

Comment:
If pdsh and libaio-dev are only required for DeepSpeed, shouldn't they be inside the conditional as well? Also, I have a preference for using the -n operator in this conditional. BASH can always do with being more explicit.

Reply:
Or perhaps this conditional should be in dockerfile.

Reply:
i've moved the conditional to top level dockerfile.